src/server/index.spec.tsx

```diff
@@ -341,6 +342,46 @@ describe(__filename, () => {
           expect(policy['connect-src']).toEqual(["'self'"]);
         });
       });
+
+      it('exposes the content of version.json', async () => {
+        const server = request(
+          createServer({
+            env: prodEnv as ServerEnvVars,
+            rootPath,
+          }),
+        );
+
+        const response = await server.get('/__version__');
+
+        expect(response.status).toEqual(200);
+        expect(response.header).toHaveProperty(
+          'content-type',
+          'application/json; charset=UTF-8',
+        );
+
+        // This file is in `src/server/fixtures/`.
+        const content = fs.readFileSync(path.join(rootPath, 'version.json'), {
```

Comment:
if you renamed `rootPath` to `fixturesPath` then I think this would be more clear and you wouldn't need a comment about it.

Reply:
Yes, I thought about it too tbh. I left it because it is the name of the argument in the `createServer()` function but that's confusing.